react/Alert/Alert.demo.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Alert, Card, Section } from 'seek-style-guide/react';
+import { LEVEL, TYPE } from '../Section/Section';
+
+const Container = ({ component: DemoComponent, componentProps }) => (
+  <Card style={{ width: '500px' }}>
+    <Section>
+      <DemoComponent {...componentProps} />
+    </Section>
+  </Card>
+);
+
+Container.propTypes = {
+  component: PropTypes.any,
+  componentProps: PropTypes.object.isRequired
+};
+
+export default {
+  route: '/alert',
+  title: 'Alert',
+  category: 'Typography',
```

Comment:
I don't think this component belongs under "Typography", but I'm not sure where we'd put it 🤔 

I feel like "Form" is a more accurate category, at least for now, even if it's useful in other contexts, but definitely open to other ideas here.